build/read-commit.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import json
+import os
+
+with open('commitinfo.json', 'r') as infile:
+    data = json.load(infile)
+    if 'commit' in data:
+        print(data['commit']['sha'])
+    else:
+        print(data)
+        exit(1)
```

Comment:
```suggestion
        sys.exit(f'Unexpected response from external-resources branch:\n {data}')
```
When I tested locally by changing the key in the if statement, the print statement didn't print anything because exit was called before it flushed. Is this better?